README.md

```diff
@@ -1638,6 +1638,14 @@ Specifies the name of the slot to create. Must be a valid replication slot name.
 
 This is the namevar.
 
+##### `ensure`
+
+Specifies the action to create or destroy named slot. Must be present
```

Comment:
Can you remove "Must be present" here and add "Required." as a line under the ensure title please?